lib/getDigest.test.js

```diff
@@ -0,0 +1,76 @@
+'use strict'
+
+const { test } = require('tap')
+const getDigest = require('./getDigest')
+
+test('Constructs a matching Digest header from a valid request Digest header and body', async ({
```

Comment:
I like the TDD approach here. We can probably dry these tests up a bit by creating the requests objects with a factory function. (just pass in the digest)

```js
function createRequest(digest) {
  return {
    headers: {
      digest
    },
    body: { hello: 'world' }
  }
}
```

Reply:
Good suggestion. Completed.